browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -65,12 +65,16 @@ class MarkdownNoteDetail extends React.Component {
   findTitle (value) {
     let splitted = value.split('\n')
     let title = null
+    let isMarkdownInCode = false
 
     for (let i = 0; i < splitted.length; i++) {
       let trimmedLine = splitted[i].trim()
-      if (trimmedLine.match(/^# .+/)) {
-        title = trimmedLine.substring(1, trimmedLine.length).trim()
-        break
+      if (trimmedLine.match('```')){
+        isMarkdownInCode = !isMarkdownInCode
+      } else if (isMarkdownInCode === false && trimmedLine.match(/^# +/)) {
+          title = trimmedLine.substring(1, trimmedLine.length).trim()
+          break
+        }
       }
     }
```

Comment:
This code doesn't work. Because there is an extra `}`. And also the indents are not good :fearful:

```
if (trimmedLine.match('```')){
  isMarkdownInCode = !isMarkdownInCode
} else if (isMarkdownInCode === false && trimmedLine.match(/^# +/)) {
  title = trimmedLine.substring(1, trimmedLine.length).trim()
  break
}
```